login.css

```diff
@@ -131,17 +137,24 @@ p{
 	margin-top:250px;
 	font-size:24px;
 	color:#0C59CF;
-	font-weight:bold;
+	font-family: 'Roboto', sans-serif;
+	font-weight:500;
 }
 
 #us{
 	height:30px;
 	width: 240px;
+	font-family: 'Roboto', sans-serif;
```

Comment:
13/08/2021: I set the fonts for the text in the username box to roboto and because it is normal text I set the font weight to 300.